test/color/test_ycbcr.py

```diff
@@ -17,6 +17,13 @@ def test_cardinality(self, device, dtype, shape):
         img = torch.ones(shape, device=device, dtype=dtype)
         assert kornia.color.rgb_to_ycbcr(img).shape == shape
 
+    @pytest.mark.parametrize("shape", [(3, 4, 4), (2, 3, 4, 4), (2, 2, 3, 4, 4)])
```

Comment:
Should 22344 success or fail? It's not an image with 5 dims, right?

Reply:
success. 5 dims is for volumetric tensors (say a video of `BTCHW`). basically this method only multiplies the channels by scalars, so it would work on any tensor with shape `(*, C, H, W)`.

Reply:
For volumetric data, it shall be `BCDHW`, so that the depth is in the third axis. I reckon we shall only support image types at the moment.

Reply:
ok, so it will work
do you want me to remove the check for 5d inputs?

Reply:
@shijianjian fixed